lib/Bailador/Request.pm

```diff
@@ -25,40 +27,61 @@ class Bailador::Request {
     }
     multi method params ($source) {
         my %ret;
+        my $source_data = '';
```

Comment:
I spoke to psch on #perl6 and he said `.chars` is right to trigger a warning, since it's counting something that is just not there (not empty, simply not even defined). I kind of agree with that, which led me to change these lines in commit e508dad to try and be a little more idiomatic, checking for defined-ness instead :)